app/views/pages/security.html.erb

```diff
@@ -0,0 +1,142 @@
+      reporting security issues</a> with others' gems. If it's a security issue
+    with the Ruby on Rails framework, see the <a href="http://rubyonrails.org/security/">
+      Rails Security</a> guide.
+  </p>
+
+  <p>
+    <strong>
+      For any security bug or issue with the RubyGems client or
+      RubyGems.org service, please email <a href="mailto:security@rubygems.org">
+        security@rubygems.org</a> with details about the problem.
+    </strong>
+  </p>
+
+  <p>
+    <small>
+      Please note: the <a href="https://groups.google.com/forum/#!forum/rubygems-developers">
+        rubygems-developers mailing list</a>, the
+      <a href="https://groups.google.com/forum/#!forum/rubygems-org">rubygems.org
+        mailing list</a>, and the <a href="ircs://irc.freenode.net:6697/#rubygems">
+        #rubygems</a> IRC channel are public areas.
+
+      If escalating to these places, please do not discuss your issue,
+      simply say that you’re trying to get a hold of someone from the security
+      team. Thanks in advance for responsibly disclosing your security issue.
+    </small>
+  </p>
+
+  <h2>Your Security</h2>
+
+  <h3>Securely pushing Gems</h3>
+  <p>
+    You may wish to <a href="http://guides.rubygems.org/security/#building-gems">
+      sign your gems</a> before releasing them to RubyGems.org.
+  </p>
+
+  <h3>Securely downloading Gems</h3>
+  <p>
+    The SHA256hash provided with files on RubyGems.org exists only to provide some
+    download corruption protection. It is not intended to provide any sort of
+    security regarding tampering. Please use GPG signing for that.
+  </p>
+
+  <p>
+    Also, see <a href="https://gist.github.com/bf4/4223e83e8becacfb2a8e">this gist</a>.
+  </p>
+
+  <h2>Reporting RubyGems.org Website Problems</h2>
+  <p>
+    If you're having trouble pushing a gem, or otherwise need help with your
+    RubyGems.org account, please <a href="http://help.rubygems.org">
+      open a new help issue</a>.
+  </p>
+
+  <p>
+    For bugs or other problems with RubyGems.org, please use the
+    <a href="https://github.com/rubygems/rubygems.org">RubyGems.org issue tracker</a>
+    to open a new issue.
+  </p>
+
+  <h2>Disclosure Policy</h2>
+
+  <p>
+    RubyGems and RubyGems.org follow a 5 step disclosure policy:
+  </p>
+
+  <ol>
+    <li>
+      Security report received and is assigned a primary handler. This person
+      will coordinate the fix and release process.
+    </li>
+    <li>
+      Problem is confirmed and, a list of all affected versions is determined.
+      Code is audited to find any potential similar problems.
+    </li>
+    <li>
+      Fixes are prepared for all releases which are still supported. These fixes
+      are not committed to the public repository but rather held locally pending
+      the announcement.
+    </li>
+    <li>
+      A suggested embargo date for this vulnerability is chosen and
+      distros@openwall is notified. This notification will include patches
```

Comment:
This is the first I've heard of distros@openwall. I am very new to infosec stuff. I think this is still just cargo-culting from other guides and I am not sure what we want to do yet.

I think we'll have to default to more of the Ruby standard security practices.